packages/components/CHANGELOG.md

```diff
@@ -46,6 +46,7 @@
 -   Normalized label line-height and spacing within the `ToolsPanel` ([36387](https://github.com/WordPress/gutenberg/pull/36387))
 -   Remove unused `reakit-utils` from peer dependencies ([#37369](https://github.com/WordPress/gutenberg/pull/37369)).
 -   Update all Emotion dependencies to the latest version to ensure they work correctly with React types ([#37365](https://github.com/WordPress/gutenberg/pull/37365)).
+-   Revert `is12Hour` condition to show `MM-DD-YYYY` when set to `true` ([#37465](https://github.com/WordPress/gutenberg/pull/37465))
```

Comment:
Let's not use the term "revert", as we're not undoing any changes — maybe something like

```suggestion
-   `DateTimePicker`: Fix the date format associated to the `is12Hour` prop ([#37465](https://github.com/WordPress/gutenberg/pull/37465))
```